server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -270,19 +266,16 @@ export default class BulkChangeModal extends React.PureComponent {
     </div>
   );
 
-  renderAssigneeOption = (option /*: { avatar?: string, email?: string, label: string } */) => (
-    <span>
-      {option.avatar != null && (
-        <Avatar
-          className="little-spacer-right"
-          hash={option.avatar}
-          name={option.label}
-          size={16}
-        />
-      )}
-      {option.label}
-    </span>
-  );
+  renderAssigneeOption = (option /*: { avatar?: string, email?: string, label: string } */) => {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove parentheses around the parameter of this arrow function. ([javascript:S3524](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fgreg%2Fhardening%2FSONAR-10298&issues=AWFHvAyvUvOPqYRWG8zq&open=AWFHvAyvUvOPqYRWG8zq)

[ISSUE_KEY]: AWFHvAyvUvOPqYRWG8zq

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED